cms/middleware/toolbar.py

```diff
@@ -47,13 +47,34 @@ def toolbar_plugin_processor(instance, placeholder, rendered_content, original_c
     return output
 
 
+def get_client_ip(request):
```

Comment:
I agree. Let's be consistent with upstream: ``HTTP_X_FORWARDED_FOR`` and ``REMOTE_ADDR`` are the standard ones, and you have to comply with those for a lot of different reasons anyway. Maybe adding the header used to check those can help clarify any issue users might have in not standard setups. In all cases it's just a matter of adding a configuration variable to the webserver to use the correct header